<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatanChooseResource</class>
 <widget class="QDialog" name="CatanChooseResource">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWool">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Wool</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonWool">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGrain">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Grain</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonGrain">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLumber">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Lumber</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonLumber">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrick">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Brick</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonBrick">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOre">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Ore</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonOre">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
